style.css

```diff
@@ -0,0 +1,131 @@
+/* Edit */
+
+h1~p {
```

Comment:
I think it's better to build standard rules and use an ID or class wherever you want to override, otherwise you might not know that the reason for the sibling styling is because you are styling a hero section. It makes the code less readable.